src/encoded/tests/datafixtures.py

```diff
@@ -549,7 +549,8 @@ def publication_data(testapp, lab, award):
 def annotation_dataset(testapp, lab, award):
     item = {
         'award': award['@id'],
-        'lab': lab['@id']
+        'lab': lab['@id'],
+        'annotation_type': 'imputation'
```

Comment:
Don't change this unless you have to fix the branch in some other way,

It's more standard to add a trailing comma to lists like this.
```
   item = {
        'award': award['@id'],
        'lab': lab['@id'],
        'annotation_type': 'imputation',
}
```